Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -494,7 +494,7 @@ public void FilterData()
         /// </summary>
         /// <param name="fieldName">Cell field name.</param>
         /// <param name="value">New cell value.</param>
-        private void UpdateCell( string fieldName, object value )
+        public void UpdateCell( string fieldName, object value )

```

Comment:
If this is public? Can I miss use this? This should only be used when the Grid is in EditMode right? If that is the case, maybe do nothing if grid is not in EditMode.

Reply:
It's public. Refresh page :)

Reply:
Well but if it's public. Can't I miss use it? Like mentioned? Or could I have a way to select the row(item) to update?

Reply:
Sorry, I was too quick to write before reading your message. I have fixed and it cannot be updated or read if not in edit state,

Reply:
Is it worth having an overload for actually updating / reading a row cell's value? Any benefit to this?
We might not want to directly update/mutate the row cell values though. It won't go trough the edit pipeline... But I guess that could be up to the developer... what do you think?

Reply:
Actually nvm, this just seems dumb.. the only way to know the row is through the item, right? If dev has the item, he can just read / mutate it directly.

Reply:
Yep. The dev can already mutate it even though it's not the right way.